### ISSUE TITLE
Iframes for uploaded videos on Publications content type have no accessible name

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12665]
## Status
- **[2024-07-08] [Fran]** Message to Laura F asking if this was truly supposed to be Iceboxed, or if it should be fixed.

## Description
On the Publications content type, the editor has the option to add an uploaded video. When added, the video generates an `iframe` on the view & edit node. However, the `iframe` is missing an accessible name because it does not have a `title` attribute. Resolving this was already investigated in a previous ticket (#11089) and we can modify this and add the `title` attribute by using the `hook_preproccess` function. Ideally the title attribute would pull the media name of the uploaded video.

Example page: https://prod.cms.va.gov/outreach-and-events/va-employment-services

### Screenshots
**View node:**
![image](https://user-images.githubusercontent.com/106678594/219678836-a37bd8c8-415c-441b-82ba-d29fa3a834df.png)
**Edit node:**
![image](https://user-images.githubusercontent.com/106678594/219678968-85f71310-a076-413a-8d29-bc1d62a47854.png)

## Acceptance Criteria
- [ ] Add a title attribute to the iframe on the view node
- [ ] Add a title attribute to the iframe on the edit node
- [ ] Use the media name of the video to populate the title attribute
- [ ] Accessibility review

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by productmike** [comment]
@wesrowe, does PW own the publications content type? The CMS team had this in refinement the other day and wanted to clarify the rightful owner

cc: @jilladams

**Comment by wesrowe** [comment]
@productmike, I suspect this is related to the outreach-materials product, which is actually going to be deprecated in q3 in favor of a new site VA built.

@laflannery, do I understand correctly that the iframe problem is in the Drupal UI, no the resulting page?

**Comment by laflannery** [comment]
@wesrowe Yes this is in the Drupal UI. If this is going to be deprecated does that mean the Drupal part will be removed/deprecated as well?

**Comment by wesrowe** [comment]
@laflannery, yes, Drupal as well. Just heard the target is end of Q3/early Q4, the other system is being built currently

**Comment by laflannery** [comment]
@wesrowe and @productmike if that's the case I would say we that we don't need to put effort into resolving this. I do think I would like to keep it open and hold onto it just until it's deprecated officially though - just to be safe if that's ok?

**Comment by wesrowe** [comment]
I added an epic and a couple labels to make it more findable in the PW world

**Comment by laflannery** [comment]
Thanks @wesrowe! I took off the CMS label since we did determine that technically it's not theirs and this way it won't keep popping up on their boards. I also added the `Blocked` label because then I'll quickly know that I don't need to bring this up as a task for folks to look at. But if you would rather me not use that label here just let me know.

**Comment by productmike** [comment]
@laflannery @wesrowe Sold. Appreciate y'all working through this!

**Comment by FranECross** [comment]
@laflannery This ticket landed in the Icebox, but I'm not sure it was supposed to. Will you please let me know if this should still be worked on? Thanks!

**Comment by laflannery** [comment]
I was told this was being deprecated so work on this shouldn't be prioritized but maybe that's no longer the case

**Comment by FranECross** [comment]
Closing as a no op
